### ISSUE TITLE
Can't login using github account - account progress

### ISSUE THREAD

**Issue by mohagas** [freeCodeCamp/freeCodeCamp#17573]
I am unable to login to FCC using my github username. I was prompted to input an email and I used the email associated with my github account. But somehow it showed me a blank account and it looks like I lost all of my progress that I worked on for almost 1 year!!!! Please assist with them ASAP

Thank you

* Browser Name: Chrome
* Browser Version: Chrome 67
* Operating System: Mac/Windows

**Comment by raisedadead** [comment]
Hi @mohagas can you please reach out to me over an email `mrugesh at freecodecamp dot org`

**Comment by raisedadead** [comment]
Okay.. you created an duplicate account accidentally. I have removed the duplicate. Please visit https://www.freecodecamp.org/deprecated-signin and try again after 5 minutes. 

You should be able to add the email that you would want and sign in.

**Comment by bmandl** [comment]
I have a similar problem, that I described in freecodecamp forum: [https://www.freecodecamp.org/forum/t/all-progress-lost-after-logging-in-with-github-instead-of-facebook/296963/8](url).
So, I had a account, connected to Facebook, but oneday I logged in with GitHub (added account). From then on, whether I log in with Facebook or with GitHub or email, I don't see my progress and certifications. Can someone fix this?
